README.md

```diff
@@ -20,6 +20,8 @@ The **ComputerManagementDsc** module contains the following resources:
 - **VirtualMemory**: allows configuration of properties of the paging file on
   the local computer.
 - **PowerShellExecutionPolicy**: Specifies the desired PowerShell execution policy.
+- **WinEventLog**: This resource allows configuration of the Logsize and Logmode
+of a specified Eventlog.
```

Comment:
@Outek, If you indent this line by 2 chars it will resolve the last error you have. The Markdown validator thinks this is a new paragraph and not part of the list and so thinks you need to have a blank line after the end of the list.